From bca4fe362cb2198f964d33c5a0fb27298d8e9ad8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 21 Nov 2022 20:03:08 +0100
Subject: [PATCH] manager: fix format strings for trigger metadata

Fixup for c8bc7519c888a99134f88f8c82353246d3c0cc5d.

(cherry picked from commit 6457ce15be84cf3c304d1ba47b89bacc2f60bf6e)

Related: #2138081
---
 src/core/timer.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/core/timer.c b/src/core/timer.c
index 8bd430b931..b6810c8599 100644
--- a/src/core/timer.c
+++ b/src/core/timer.c
@@ -948,11 +948,11 @@ static int activation_details_timer_append_env(ActivationDetails *details, char
         if (!dual_timestamp_is_set(&t->last_trigger))
                 return 0;
 
-        r = strv_extendf(strv, "TRIGGER_TIMER_REALTIME_USEC=%" USEC_FMT, t->last_trigger.realtime);
+        r = strv_extendf(strv, "TRIGGER_TIMER_REALTIME_USEC=" USEC_FMT, t->last_trigger.realtime);
         if (r < 0)
                 return r;
 
-        r = strv_extendf(strv, "TRIGGER_TIMER_MONOTONIC_USEC=%" USEC_FMT, t->last_trigger.monotonic);
+        r = strv_extendf(strv, "TRIGGER_TIMER_MONOTONIC_USEC=" USEC_FMT, t->last_trigger.monotonic);
         if (r < 0)
                 return r;
 
@@ -974,7 +974,7 @@ static int activation_details_timer_append_pair(ActivationDetails *details, char
         if (r < 0)
                 return r;
 
-        r = strv_extendf(strv, "%" USEC_FMT, t->last_trigger.realtime);
+        r = strv_extendf(strv, USEC_FMT, t->last_trigger.realtime);
         if (r < 0)
                 return r;
 
@@ -982,7 +982,7 @@ static int activation_details_timer_append_pair(ActivationDetails *details, char
         if (r < 0)
                 return r;
 
-        r = strv_extendf(strv, "%" USEC_FMT, t->last_trigger.monotonic);
+        r = strv_extendf(strv, USEC_FMT, t->last_trigger.monotonic);
         if (r < 0)
                 return r;
 
